Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -828,10 +828,10 @@ void Document::adopt_node(Node& node)
 ExceptionOr<NonnullRefPtr<Node>> Document::adopt_node_binding(NonnullRefPtr<Node> node)
 {
     if (is<Document>(*node))
-        return DOM ::NotSupportedError::create("Cannot adopt_ref a document into a document");
+        return DOM::NotSupportedError::create("Cannot adopt a document into a document");
```

Comment:
I wonder why clang-format wasn't taking care of this random space (which subsequently would've failed the CI build so we notice)...